UPGRADE-1.3.md

```diff
@@ -1 +1,27 @@
 # UPGRADE FROM `v1.2.X` TO `v1.3.0`
+
+## Configuration
+
+* **Sylius 1.3** requires **PHP 7.2**, then following configuratio should be changed:
+    * `composer.json`:
+        * `"php": "^7.1"` -> `"php": "^7.2"`
+        * `"config.platform.php": 7.1.18` -> `"config.platform.php": 7.2.4`
+    * `.travis.yml` (if you're using TravisCI):
+        * leave only configuration for `PHP 7.2`
+
+* Copy `vendor/sylius/sylius/env.dist` to your application root:
```

Comment:
I'd go for `https://github.com/Sylius/Sylius-Standard/blob/1.3/.env.dist` instead of going into vendors and referencing `Sylius/Sylius`.